manticore/manticore.py

```diff
@@ -128,6 +141,10 @@ def binary_type(path):
         return 'PE'
     elif magic == '\x7fCGC':
         return 'DECREE'
+    # FIXME (theo) magic for file containing Binja IL.
+    # For now we just load the binary, this is to be implemented
+    elif magic == 'BNJA':
```

Comment:
does a file format BNJA with Binja IL exist currently? if not, maybe we should remove this until it does?